### ISSUE TITLE
pull access denied for qemu-handler, repository does not exist or may require 'docker login'

### ISSUE THREAD

**Issue by n8henrie** [Robertof/nixos-docker-sd-image-builder#29]
I keep getting `pull access denied for qemu-handler, repository does not exist or may require 'docker login'`. `docker login` doesn't help. Where is this repo? No results at <https://hub.docker.com/search?q=qemu-handler&type=image>

I had to update the `docker-compose` commands to `docker compose` (Arch Linux host), not sure if that may pose the problem.

**Comment by Robertof** [comment]
Hey. `qemu-handler` should be the name of the generated image after building the `setup-qemu` container:

<img width="912" alt="image" src="https://user-images.githubusercontent.com/59401/136105372-95b9da09-557d-4195-937d-8721f081d815.png">

Did you install the [`docker-compose`](https://archlinux.org/packages/community/x86_64/docker-compose/) package from `pacman`? It should work out of the box using `docker-compose`.

Thanks for using this!

**Comment by Robertof** [comment]
Done! @n8henrie can you try to see if the issue is fixed with the latest commit?

**Comment by n8henrie** [comment]
Running like a champ. Thank you!

**Comment by Chris2011** [comment]
Couple of hours ago, I deleted all docker stuff what was created from your repo and I got exactly the same error. I didn't has this before. It took some seconds and then the error comes up.

<img width="858" alt="image" src="https://user-images.githubusercontent.com/795658/147299768-b8175f4c-e697-4ff1-bd0c-4610fd19e7ea.png">

I run the script in Ubuntu (WSL) on Windows 11 with docker and docker-compose support. Maybe I will try to reinstall docker.

**Comment by Chris2011** [comment]
I fixed this, I accidentally removed the build folder in setup-qemu 😅

**Comment by Robertof** [comment]
Sorry for the late answer @Chris2011 - glad you sorted it out! Incidentally, your output made me realize about a potential issue with Compose version numbers which I've fixed in 3478858. Thanks!

**Comment by Chris2011** [comment]
Yes, I already saw this issue and a colleage and me fixed this with an regex. Wanted to make a PR but you were faster :). Just in case. For version < 2, there is no v in front of. I have version 1.29.2 and with version > 2 we have v2.*.*. Just to let you know. Thx for the answer.

**Comment by Robertof** [comment]
Awesome, thanks for the insights! I think now it should just work with either the 'v' or not in front - thanks for wanting to contribute as well!
